src/component/handlers/edit/editOnSelect.js

```diff
@@ -20,10 +20,8 @@ var getDraftEditorSelection = require('getDraftEditorSelection');
 import type DraftEditor from 'DraftEditor.react';
 
 function editOnSelect(editor: DraftEditor): void {
-  if (editor._blockSelectEvents) {
-    return;
-  }
-  if (editor._latestEditorState !== editor.props.editorState) {
+  if (editor._blockSelectEvents ||
```

Comment:
Actually - going to land this first to get Github in sync with our internal version, then will make a separate PR to update the conditional.